rskj-core/src/test/java/org/ethereum/net/server/ChannelManagerImplTest.java

```diff
@@ -102,12 +99,7 @@ void blockAddressCheck_logInvalidInetAddressException() throws UnknownHostExcept
         channelManager.add(peer);
         channelManager.tryProcessNewPeers();
 
-        Logger logger = mock(Logger.class);
-        TestUtils.setFinalStatic(channelManager, "logger", logger);
-
         Assertions.assertTrue(channelManager.isAddressBlockAvailable(otherPeer.getInetSocketAddress().getAddress()));
-
-        verify(logger).error(anyString(), any(Throwable.class));
```

Comment:
so now, are not we checking if a log was written?

Reply:
yes. First, there's a doubt on whether we need to cover logging with unit tests. Second, in this particular case verifying if the logger receives any string seems not to be that useful. And last but not least, starting from java17 it's prohibited to set private static field via reflection (in our case it's a logger)